src/instruments/src/EFB/Efb.tsx

```diff
@@ -162,9 +162,7 @@ const Efb = () => {
             return;
         }
 
-        console.log('Fetching simbriefData');
```

Comment:
why were these logs removed? they seem useful for debugging and wont' clog the console.

Reply:
I don't think it is necessary to annotate certain operations being done, and anything that goes wrong whilst these operations happen would preferably throw an error that could be handled and that would be the primary way of debugging in the event that something went wrong.